templates/home.html

```diff
@@ -17,56 +17,91 @@
+        <h4>Keep 100% ownership of your music</h4>
+        <p>We never take a cut of your investments or revenue. You always have 100% control over your music and can leave the platform at anytime. We created a system that is built on the integrity of our artists and community.</p>
         <br />
 
-        <p class="how-page">2. THE ARTIST GIVES A PERCENTAGE OF SALES TO PEOPLE WHO INVEST IN THE MUSIC PROJECT</p>
+        <h2>GIVE A PERCENTAGE OF SALES BACK TO PEOPLE WHO INVEST IN YOUR MUSIC PROJECT</h2>
         <br />
 
         <div id="canvas-holder">
             <canvas id="chart-area" width="500" height="500"></canvas>
         </div>
 
-        <p class="how-copy">ARTISTS KEEP FULL OWNERSHIP AND CONTROL OF THEIR MUSIC.</p>
-        <h4 class="how-p">An artist gives a percentage of sales back to the investors as an incentive for them to fund and promote their music. PerDiem takes care of all the complicated stuff (tracking, managing, and distributing funds). </h4>
+        <h3>Take full control of your music revenue</h3>
+        <p>Gives a percentage of sales back to the investors as an incentive for them to fund and promote their music. We take care of all the complicated stuff for you (distribution and liscensing, tracking, and managing funds). This allows you focus on doing what you do best... making music.</p>
         <br />
 
-        <p class="how-page">3. REVENUE IS SPLIT FROM THE PROJECT BETWEEN THE ARTIST AND THE PEOPLE WHO INVEST IN IT</p>
+        <h2>BUILD A TEAM OF PEOPLE ALL PUSHING FOR YOUR SUCCESS</h2>
         <div id="canvas-box">
             <canvas id="line-canvas"></canvas>
         </div>
-
-        <p class="how-copy">PEOPLE NOW HAVE AN INCENTIVE TO BUY AND PROMOTE THE ARTISTS MUSIC</p>
-        <h4 class="how-p">100% of revenue earned from the project is split between the artist and investors. Since the success of their investment is tied to the success of the artist, fans have an incentive to promote the music they have invested in. The more music the artist sells, the more money they can earn.</h4>
-        <br />
-        <br />
+        <h3>">GIVE PEOPLE AN INCENTIVE TO FUND AND PROMOTE YOUR MUSIC</h3>
+        <p>Since the success of the fans investment is tied to the success of the artist, fans have an incentive to promote the music they have invested in. The more music the artist sells, the more money they can earn. It's a win-win for everyone.</p>
         <br />
-
-        <p class="how-copy">WANT TO LEARN EVEN MORE?</p>
-        <p class="how-p">CHECK OUT <a href="{% url 'faq' %}">OUR FAQ</a> AND <a href="{% url 'trust' %}">TRUST &amp; SAFETY</a>
-        <br />JOIN OUR VIP LIST </p>
-        <br />
-        <!-- How it works end -->
+        <h5>Interested in being on PerDiem?</h5>
+        <div class="apply-button">
+            <a href="{% url 'artist_application' %}" class="button">Apply Here</a>
+        </div>
+    </div>
+            <!-- How it works for fans -->
```

Comment:
This comment should be unindented so that it's aligned with the `<div>` below it.